cmd/otelcontribcol/receivers_test.go

```diff
@@ -187,7 +187,8 @@ func TestDefaultReceivers(t *testing.T) {
 			receiver: "filestats",
 		},
 		{
-			receiver: "flinkmetrics",
+			receiver:      "flinkmetrics",
+			skipLifecycle: true, // Requires connecting to a running flink cluster
```

Comment:
Looks like the field has a typo `skipLifecyle` 🤦 . feel free to update the original field